templates/prescription/general_edit.html

```diff
@@ -325,7 +326,7 @@
 
     $("#drug").focus();
     $("#prescribe").submit(function() { return CheckForErrors(this) });
-    $("#druglookup").click(function() { DoDrugLookup(this) });
+    //$("#druglookup").click(function() { DoDrugLookup(this) }); // patched out until further notice. not needed!!?
```

Comment:
Is this removing functionality?

Reply:
I don't know. It does't work and hasn't worked for awhile that I can see. Appears it just popup a window with the same search div that is used by the click here for drug search. What did it use to do?

Reply:
That's odd. Let me look into this. I recall ensuring this worked when we integrated WENO rx code into codebase.

Reply:
Yep, tested this is not needed(both search mechanisms work without it).
Rec:
1. removing this line
2. removing the DoDrugLookup function
3. Replace line 14 with:
``````
{headerTemplate assets='jquery-ui|common'}
``````
And remove lines 15-17 and lines 45-48

Reply:
I don't understand what it is meant to do. Is it suppose to popup information about the drug selected or a web drug search? Seems in the past it was a web search, just not sure, but if it makes sense, i'll fix.

Reply:
Also, related to headerTemplate. If we really need to include the base jquery-ui.css here, then will add a jquery-ui-base entry in config/config.yaml

Reply:
@sjpadgett ,
Per above, the DoDrugLookup stuff can all be yanked. It appears this was improved with the embedded `(click here to search)` link below the drug name, which is an online search. Note that as I recall this item may require the base jquery-ui css stuff.

Reply:
The jquery-ui was added for weno's drug autocomplete lookup which also needs attention to bring that drug handling in line with base OpenEMR's method. I really would prefer not adding jquery-ui to auto load just because of new dialog because, I'm somewhat fearful of that library and will probably write my own resize function. Also, new dialog will add the library if not present so, up to you. The jquery-ui.css may be needed so I will test that.

Reply:
Was just placing the jquery-ui in above headerTemplate call since this script was calling it.(agree we should not be autoloading it for all Header calls)